### PR TITLE
feat: add Python 3.8 and 3.9 Docker images

### PR DESCRIPTION
**Pull request by inverse**
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules

#### What does this PR do?

Add Python 3.8 and 3.9 images. I am not sure where the build is hooked up. But I'm guessing there is some automation that splits on `.` and uses that as the tag? 

Perhaps a better versioning schema going forward would be to have the `Dockerfile.python-3` image defined as the latest Python version.  What are your thoughts on this?

#### How should this be manually tested?

From within the `/docker` directory:

```bash
docker build -t snyk/snyk-cli:python-3.8 --file Dockerfile.python-3.8 .
docker build -t snyk/snyk-cli:python-3.9 --file Dockerfile.python-3.9 .
```

### CODE REVIEW

File: docker/Dockerfile.python-3.9 [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+MAINTAINER Snyk Ltd
+
+RUN mkdir /home/node
+WORKDIR /home/node
+
+# Install Python utilities, node, Snyk CLI
+RUN pip install pip pipenv virtualenv -U && \
+    apt-get update && \
+    apt-get install -y build-essential curl git && \
+    curl -sL https://deb.nodesource.com/setup_14.x | bash - && \
```
Comment (inverse) [inline docker/Dockerfile.python-3.9:12]: All other images are using 10.x which is not recommended. When building I saw

```
  Node.js 10.x is no longer actively supported!

  You will not receive security or critical stability updates for this version.

  You should migrate to a supported version of Node.js as soon as possible.
  Use the installation script that corresponds to the version of Node.js you
  wish to install. e.g.

   * https://deb.nodesource.com/setup_12.x — Node.js 12 LTS "Erbium"
   * https://deb.nodesource.com/setup_14.x — Node.js 14 LTS "Fermium" (recommended)
   * https://deb.nodesource.com/setup_15.x — Node.js 15 "Fifteen"
   * https://deb.nodesource.com/setup_16.x — Node.js 16 "Gallium"
```

If you agree with this change I will create another PR to bring these all to the recommended 14.x

**Comment by aviadhahami** [comment]
Hey @inverse !
Thank you for the contribution 🙏

**Review by aviadhahami — APPROVED** [review]
LGTM :)

**Review by Avishagp — APPROVED** [review]

**Comment by aviadhahami** [comment]
Hey @inverse!
I'm closing this PR in favor of https://github.com/snyk/snyk/pull/2037, so circle jobs run [commit was cherry-picked]

Thank you v much for your contribution!

**Pull request closed without merging**